client/src/main/webjar/organization-details/organization-details.html

```diff
@@ -0,0 +1,145 @@
+<div ng-if="organizationDetails.loading"
+     layout="row"
+     layout-sm="column"
+     layout-align="space-around">
+    <md-progress-circular md-mode="indeterminate">
+    </md-progress-circular>
+</div>
+
+<div ng-if="organizationDetails.orgInfo"
+     layout="column"
+     layout="fill">
+
+    <!-- Toolbar with program actions. -->
+    <md-toolbar class="md-hue-2">
+        <div class="md-toolbar-tools">
+
+            <md-button class="md-icon-button"
+                       aria-label="Back to map"
+                       ui-sref="map">
+                <md-icon md-font-library="material-icons">
+                    arrow_back
+                </md-icon>
+                <md-tooltip md-direction="bottom">
+                    <span>
+                        Back to map
+                    </span>
+                </md-tooltip>
+            </md-button>
+
+            <h2>
+                <span ng-bind="organizationDetails.orgInfo.name">
+                </span>
+            </h2>
+
+            <span flex></span>
+
+            <md-button class="md-icon-button"
+                       aria-label="Delete organization"
+                       ng-click="organizationDetails.deleteOrganization($event)">
+                <md-icon md-font-library="material-icons">
+                    delete
+                </md-icon>
+                <md-tooltip md-direction="bottom">
+                    <span>
+                        Delete organization
+                    </span>
+                </md-tooltip>
+            </md-button>
+
+            <md-button class="md-icon-button"
+                       aria-label="Edit organization"
+                       ng-click="organizationDetails.edit($event)">
+                <md-icon md-font-library="material-icons">
+                    edit
+                </md-icon>
+                <md-tooltip md-direction="bottom">
+                    <span>
+                        Edit organization
+                    </span>
+                </md-tooltip>
+            </md-button>
+
+        </div>
+    </md-toolbar>
+
+    <md-content flex>
+
+         <!-- Organization information card -->
+
+        <md-card>
+
+            <md-card-title>
+                <md-card-title-text>
+                    <span class="md-headline"
+                          md-colors="{color: 'primary'}"
+                          ng-bind="organizationDetails.orgInfo.name">
+                    </span>
+                </md-card-title-text>
+            </md-card-title>
+
+            <md-card-content layout="column">
```

Comment:
Isn't this the same as the organization card in the program details?

You can merge this like this, but I will follow behind you and extract a new organization-card component for this instead of copy/pasting it.